src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,164 @@
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin(CORRECT_LOGIN);
+        user.setAge(CORRECT_AGE);
+        user.setPassword(CORRECT_PASSWORD);
+    }
+
+    @Test
+    void addCorrectUsers_Ok() {
+        Assertions.assertEquals(user, registrationService.register(user));
+        user = new User();
+        user.setLogin("Login1234");
+        user.setPassword(CORRECT_PASSWORD);
+        user.setAge(CORRECT_AGE);
+        Assertions.assertEquals(user, registrationService.register(user));
+        int expectedSize = 2;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addUserWithLoginEdgeLength_Ok() {
+        user.setLogin("Login1");
+        Assertions.assertEquals(user, registrationService.register(user));
+        user = new User();
+        user.setLogin("Login12");
+        user.setPassword(CORRECT_PASSWORD);
+        user.setAge(CORRECT_AGE);
+        Assertions.assertEquals(user, registrationService.register(user));
+        int expectedSize = 2;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addUserWithLoginLessSix_NotOk() {
+        user.setLogin("Log");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(user));
+        user.setLogin("Login");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(user));
+        int expectedSize = 0;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addExistingUser_NotOk() {
+        registrationService.register(user);
+        User existingUser = new User();
+        existingUser.setId(user.getId());
+        existingUser.setLogin(user.getLogin());
+        existingUser.setPassword(user.getPassword());
+        existingUser.setAge(user.getAge());
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(existingUser));
+        int expectedSize = 1;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addUserWithPasswordLessSix_NotOk() {
+        user.setPassword("12345");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(user));
+        user.setPassword("abc");
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(user));
+        int expectedSize = 0;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addUserWithPasswordEdgeLength_Ok() {
+        user.setPassword("123456");
+        Assertions.assertEquals(user, registrationService.register(user));
+        int expectedSize = 1;
+        Assertions.assertEquals(expectedSize, Storage.people.size());
+    }
+
+    @Test
+    void addNullUser() {
```

Comment:
use the same name format 
```suggestion
    void addNullUser_NotOk() {
```
check it everywhere